### ISSUE TITLE
Respect "Dynamic Type" (Default font size) for Compose view

### ISSUE THREAD

**Issue by lima46zulu** [damus-io/damus#725]
The text in the compose window (new or replies) is one of the only text element in the app that doesn't respect the dynamic type setting. Currently the text in the compose window is too small. 

- Composing new notes
- Writing a reply

**Comment by bryanmontz** [comment]
Fixed in PR https://github.com/damus-io/damus/pull/761

**Comment by bryanmontz** [comment]
@alltheseas This Issue can be closed now because the PR was merged.

**Comment by bryanmontz** [comment]
You're right. It makes sense to wait until you can test it in a build. Thanks for all your help!

**Comment by alltheseas** [comment]
Closing as part of a backlog triage — we're narrowing focus to the most actionable issues to reduce noise for contributors.

If this is still relevant, please reopen with updated context or reproduction steps. Thanks for reporting!
